test/jdk/sun/net/www/URLConnectionTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.net.URL;
+
+/*
+ * @test
+ * @bug 8252767
+ * @summary Tests sun.net.www.URLConnection
+ * when already connected
+ * @run testng URLConnectionTest
+ */
+public class URLConnectionTest {
```

Comment:
I think it would be better to put the test in test/jdk/java/net/URLConnection as its a test for URLConnection.setRequestProperty from the API user point of view. Also it's a a specific test for the behaviour of one method so I think needs a more specific name too ("URLConnectionTest" is too general)

Reply:
Hello Alan,

There's a `test/jdk/java/net/URLConnection/RequestProperties.java` test class which tests the `setRequestProperty` for `NullPointerException` expectations of that API. Perhaps it would be OK to add this test into that class?

Reply:
I've now updated this PR to move the test into `test/jdk/java/net/URLConnection/RequestProperties.java`